pkg/syncer/util.go

```diff
@@ -180,3 +180,15 @@ func HasMigratedToAnnotationUpdate(ctx context.Context, prevAnnotations map[stri
 	log.Debugf("%v annotation not found", common.AnnMigratedTo)
 	return false
 }
+
+// migratedToAnnotationExists returns true if the migrated-to annotation is found for a given PV
+func migratedToAnnotationExists(ctx context.Context, annotations map[string]string) bool {
+	log := logger.GetLogger(ctx)
+	// Checking if the migrated-to annotation is found in the PV
+	if _, annMigratedToFound := annotations[common.AnnMigratedTo]{
```

Comment:
Do we want to check `pv.kubernetes.io/migrated-to` is set to `csi.vsphere.vmware.com`?

Reply:
Yes!